### PR TITLE
[Not to Merge] Natspec2

### PR DESCRIPTION
**Pull request by OdedWx**
Natspec comments

### CODE REVIEW

File: contracts/spec_interfaces/ICommittee.sol [hunk 1]
```diff
@@ -13,32 +13,61 @@ interface ICommittee {
...
+	/// Returns data on the currently appointed committee
+	/// @return generalCommitteeSize - the number of members in the committee
+	/// @return certifiedCommitteeSize - the number of certified members in the committee
+	/// @return totalWeight - the total effective stake / weight of the committee
+	function getCommitteeStats() external view returns (uint generalCommitteeSize, uint certifiedCommitteeSize, uint totalWeight);
+
+	/// Returns data on a committee member
+	/// @param addr - the committee member address
+	/// @return inCommittee - indicates wether the queried address is a member in the committee
+	/// @return weight - the the committee member weight
```
Comment (ordd) [inline contracts/spec_interfaces/ICommittee.sol:64]: typo

File: contracts/spec_interfaces/IDelegation.sol [hunk 2]
```diff
@@ -15,17 +15,40 @@ interface IDelegations /* is IStakeChangeNotifier */ {
...
+	/// @param addr is the address to refresh its stake
 	function refreshStake(address addr) external /* onlyWhenActive */;
 
+	/// Returns the delegated stake of an addr 
+	/// @dev an address that is not self delegating has a 0 delegated stake
+	/// @param addr is the address to query
+	/// @return delegatedStake is the address delegated stake
 	function getDelegatedStake(address addr) external view returns (uint256);
 
+	/// Returns the delegated stake of an addr
```
Comment (ordd) [inline contracts/spec_interfaces/IDelegation.sol:36]: unclear
it returns the delegate address of the given delegator

File: contracts/spec_interfaces/IDelegation.sol [hunk 3]
```diff
@@ -36,7 +59,19 @@ interface IDelegations /* is IStakeChangeNotifier */ {
...
+	/// @param from is a list of delegator addresses.
+	/// @param to - the address the the delegators delegate to.
 	function importDelegations(address[] calldata from, address to) external /* onlyMigrationManager onlyDuringDelegationImport */;
 
+	/// Imports delegations during initial migration, operates 
+	/// @dev initialization function called only by the initializationManager.
+	/// @dev Does not update the Rewards or Election contracts.
+	/// @dev assumes deactivated Rewards.
+	/// @param from is a list of delegator addresses.
+	/// @param to - the address the the delegators delegate to.
```
Comment (ordd) [inline contracts/spec_interfaces/IDelegation.sol:75]: typo

File: contracts/spec_interfaces/IElections.sol [hunk 4]
```diff
@@ -19,31 +19,59 @@ interface IElections {
...
 	 */
 
-	/// @dev Called by a guardian when ready to start syncing with other nodes
+	/// Called by a guardian when ready to start syncing with other nodes
+	/// @dev ready to sync state is not managed in the contract that only emits an event
+	/// @dev readyToSync clears the readyForCommittee state
 	function readyToSync() external;
 
-	/// @dev Called by a guardian when ready to join the committee, typically after syncing is complete or after being voted out
+	/// Called by a guardian when ready to join the committee, typically after syncing is complete or after being voted out
```
Comment (ordd) [inline contracts/spec_interfaces/IElections.sol:27]: after being voted out > after being voted unready

File: contracts/spec_interfaces/IElections.sol [hunk 5]
```diff
@@ -19,31 +19,59 @@ interface IElections {
...
+	/// @return certification is a list of bool indicating the committee members certification
+	/// @return ips is a list of the committee members' ip
 	function getCommittee() external view returns (address[] memory committee, uint256[] memory weights, address[] memory orbsAddrs, bool[] memory certification, bytes4[] memory ips);
 
 	// Vote-unready
 
+	/// Casts an unready vote on a subject guardian
 	/// @dev Called by a guardian as part of the automatic vote-unready flow
+	/// @dev The transaction may be sent from the guardian or orbs address.
+	/// @param subject is the subject guardian to vote out
```
Comment (ordd) [inline contracts/spec_interfaces/IElections.sol:55]: to vote out > to vote unready

File: contracts/spec_interfaces/IElections.sol [hunk 6]
```diff
@@ -92,35 +135,47 @@ interface IElections {
...
-	/// @dev Returns the minimum self-stake required for the effective stake
+	/// Returns the minimum self-stake required for the effective stake
+	/// @return minSelfStakePercentMille is the minimum self stake in percent-mille (0-100,000)
 	function getMinSelfStakePercentMille() external view returns (uint32);
 
-	/// @dev Sets the vote-out threshold
-	/// voteOutPercentMilleThreshold - the minimum threshold in percent-mille (0-100,000)
+	/// Sets the vote-out threshold
+	/// @dev governance function called only by the functional manager
+	/// @param voteUnreadyPercentMilleThreshold is the minimum threshold in percent-mille (0-100,000)
```
Comment (ordd) [inline contracts/spec_interfaces/IElections.sol:149]: voteUnreadyPercentMilleThreshold > voteOutPercentMilleThreshold

File: contracts/spec_interfaces/IElections.sol [hunk 7]
```diff
@@ -92,35 +135,47 @@ interface IElections {
...
 	function setVoteOutPercentMilleThreshold(uint32 voteUnreadyPercentMilleThreshold) external /* onlyFunctionalManager onlyWhenActive */;
```
Comment (ordd) [inline contracts/spec_interfaces/IElections.sol:150]: voteUnreadyPercentMilleThreshold > voteOutPercentMilleThreshold

File: contracts/spec_interfaces/ICommittee.sol (hunk 1 above)
Comment (OdedWx) [reply contracts/spec_interfaces/ICommittee.sol:64]: done

File: contracts/spec_interfaces/IDelegation.sol (hunk 2 above)
Comment (OdedWx) [reply contracts/spec_interfaces/IDelegation.sol:36]: done

File: contracts/spec_interfaces/IDelegation.sol (hunk 3 above)
Comment (OdedWx) [reply contracts/spec_interfaces/IDelegation.sol:75]: done

**Pull request closed without merging**